code/game/gamemodes/miniantags/guardian/types/healer.dm

```diff
@@ -11,8 +15,9 @@
 	var/obj/effect/bluespace_beacon/beacon
 	var/beacon_cooldown = 0
 	var/default_beacon_cooldown = 300 SECONDS
-	var/toggle = FALSE
+	var/toggle = COMBAT
```

Comment:
```suggestion
	var/stance = COMBAT
```
`stance` or whatever makes more sense as a name than `toggle`

Reply:
Can change it, all other guardians use toggle, though this has 3 modes